app/helpers/admin/claims_helper.rb

```diff
@@ -0,0 +1,24 @@
+module Admin
+  module ClaimsHelper
+    include ::ClaimsHelper
+
+    def admin_eligibility_answers(eligibility)
+      [].tap do |a|
+        a << [t("student_loans.questions.admin.qts_award_year"), academic_years(eligibility.qts_award_year)]
+        a << [t("student_loans.questions.admin.claim_school"), eligibility.claim_school_name]
+        a << [t("questions.admin.current_school"), eligibility.current_school_name]
+        a << [t("student_loans.questions.admin.subjects_taught"), subject_list(eligibility.subjects_taught)]
+        a << [t("student_loans.questions.admin.had_leadership_position"), (eligibility.had_leadership_position? ? "Yes" : "No")]
+        a << [t("student_loans.questions.admin.mostly_performed_leadership_duties"), (eligibility.mostly_performed_leadership_duties? ? "Yes" : "No")] if eligibility.had_leadership_position?
+      end
+    end
+
+    def admin_identity_answers(claim)
+      [].tap do |a|
```

Comment:
Given there are no conditionals in this code, does it make sense to make this simply return an array without the `tap` and append calls?